volatility3/framework/symbols/linux/extensions/__init__.py

```diff
@@ -2673,7 +2402,9 @@ def get_inode_type(self) -> Union[str, None]:
         else:
             return None
 
-    def _time_member_to_datetime(self, member) -> datetime.datetime:
+    def _time_member_to_datetime(
+        self, member
+    ) -> Union[datetime.datetime, interfaces.renderers.BaseAbsentValue]:
```

Comment:
I noticed that the renders are now being used in the core API, whereas previously, they were only used in the plugins final stage (mostly in the _generator or around it). Doesn't that seem a bit unusual when the framework is used as a library?